vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1037,15 +1064,68 @@ public void run() {
         mSearchView.setIconifiedByDefault(false);
         mSearchView.setOnQueryTextListener(this);
 
-        if (null != mFloatingActionButton) {
-            mFloatingActionButton.setOnClickListener(new View.OnClickListener() {
+        mFabStartChat.setIconDrawable(ThemeUtils.INSTANCE.tintDrawableWithColor(
+                getResources().getDrawable(R.drawable.ic_person_black_24dp), getResources().getColor(android.R.color.white)
+        ));
+
+        mFabCreateRoom.setIconDrawable(ThemeUtils.INSTANCE.tintDrawableWithColor(
+                getResources().getDrawable(R.drawable.ic_add_white), getResources().getColor(android.R.color.white)
+        ));
+
+        mFabJoinRoom.setIconDrawable(ThemeUtils.INSTANCE.tintDrawableWithColor(
+                getResources().getDrawable(R.drawable.riot_tab_rooms), getResources().getColor(android.R.color.white)
+        ));
+
+        // Pre-Lollipop does not support elevation so cannot have touch guard
+        // above bottomNavigationView.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
The touch guard is not shown on PreLollipop, it is a bit annoying. If there is no elevation before API 21, I think this is not a problem to show the touchguard.

Reply:
The only reason why I've disabled the touch guard pre-lollipop is because the touch guard shows up behind the Bottom Navigation View. No matter the order of views in the constraint layout, the Bottom Navigation View shows up on top of everything.
Would it be okay to have the Bottom Navigation View not be covered by the touch guard?

Reply:
Ok, so it means that the Bottom Navigation View is able to use elevation, maybe by using app: namespace. If it is not possible to do better, yes, I think it is better to have the touch guard in all cases